### PR TITLE
Circle build

### PR DESCRIPTION
**Pull request by geokala**

### CODE REVIEW

**Review by AdarShaked — CHANGES_REQUESTED** [review]
pls add also this file :
https://github.com/cloudify-cosmo/cloudify-gcp-plugin/blob/master/.circleci/package_release.py

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    docker:
+      - image: circleci/python:3.6
+
+  wagon_generator:
+    machine:
+      image: ubuntu-1604:201903-01
+
+
+commands:
+  run_unittest_py27:
```
Comment (AdarShaked) [inline .circleci/config.yml]: unitests should run on py36 too?

Comment (geokala) [reply .circleci/config.yml]: There aren't any at the moment I think, sorry- forgot to remove the command as well as the job.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            branches:
+              only: /([0-9\.]*\-build|master|dev)/
+      - release:
+          filters:
+            branches:
+              only: /master/
+          requires:
+            - wagon
+            - rhel_wagon
+            - integration_tests
```
Comment (AdarShaked) [inline .circleci/config.yml]: remove that, there is   no "integration tests":)

File: .circleci/config.yml (hunk 1 above)
Comment (geokala) [reply .circleci/config.yml]: (the tests are openstack specific at the moment)

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+      - rhel_wagon:
+          filters:
+            branches:
+              only: /([0-9\.]*\-build|master|dev)/
+      - release:
+          filters:
+            branches:
+              only: /master/
+          requires:
+            - wagon
```
Comment (AdarShaked) [inline .circleci/config.yml:133]: add "unittests_py27" pls . missed that before.

**Review by AdarShaked — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (geokala) [reply .circleci/config.yml]: (removed this)

File: .circleci/config.yml (hunk 2 above)
Comment (geokala) [reply .circleci/config.yml]: Done

File: .circleci/config.yml (hunk 3 above)
Comment (geokala) [reply .circleci/config.yml:133]: I removed unittests_py27 as explained above (it currently just has integration tests and they're openstack specific)

**Review by AdarShaked — APPROVED** [review]

**Pull request merged**